### ISSUE TITLE
material-ui missing SVG icon definitions?

### ISSUE THREAD

**Issue by RichardJECooke** [DefinitelyTyped/DefinitelyTyped#7189]
I see the material-ui.d.ts file has only three svg-icons definitions:

```
declare module 'material-ui/lib/svg-icons/navigation/menu' {
    import NavigationMenu = __MaterialUI.NavigationMenu;
    export = NavigationMenu;
}

declare module 'material-ui/lib/svg-icons/navigation/chevron-left' {
    import NavigationChevronLeft = __MaterialUI.NavigationChevronLeft;
    export = NavigationChevronLeft;
}

declare module 'material-ui/lib/svg-icons/navigation/chevron-right' {
    import NavigationChevronRight = __MaterialUI.NavigationChevronRight;
    export = NavigationChevronRight;
}
```

Are all the other icons still requested features, or are they left out deliberately and I'm doing something wrong trying to use them like this:

```
import MoreVertIcon = require('material-ui/lib/svg-icons/navigation/more-vert');
import CallEnd = require('material-ui/lib/svg-icons/communication/call-end');
import CallMade = require('material-ui/lib/svg-icons/communication/call-made');
import CallMerge = require('material-ui/lib/svg-icons/communication/call-merge');
```

**Comment by RichardJECooke** [comment]
I forked the repo, made a branch called issue7189, added a definition for more-vert icon, uploaded it, and made a pull request here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/7190.

Basically I just copied and pasted the code for the other icons and it seemed to work for one of the many missing ones.  I've never made a pull request before so please let me know if I did anything wrong.

**Comment by siavelis** [comment]
@RichardJECooke please close this issue, your pull request was merged.

**Comment by andy-ms** [comment]
Fixed by #18931.
